warehouse/scripts/dbt_artifacts/__init__.py

```diff
@@ -151,43 +154,54 @@ class HookNode(BaseHookNode, NodeModelMixin):
 
 class ModelNode(BaseModelNode, NodeModelMixin):
     @property
-    def gvrepr(self) -> str:
-        if (
-            self.config
-            and self.config.materialized in ("table", "incremental")
-            and self.catalog_entry
-            and self.catalog_entry.num_bytes  # type: ignore[attr-defined]
-        ):
-            return "\n".join(
-                [
-                    super(ModelNode, self).gvrepr,
-                    f"Storage: {humanize.naturalsize(self.catalog_entry.num_bytes)}",  # type: ignore[attr-defined]
-                ]
-            )
-        return super(ModelNode, self).gvrepr
+    def children(self) -> List["ModelNode"]:
+        children = []
+        for unique_id, node in NodeModelMixin._instances.items():
+            if (
+                isinstance(node, ModelNode)
+                and node.depends_on
+                and node.depends_on.nodes
+                and self.unique_id in node.depends_on.nodes
+            ):
+                children.append(node)
+
+        return children
 
     @property
     def gvattrs(self) -> Dict[str, Any]:
-        fillcolor = "white"
-
-        if self.config and self.config.materialized in ("table", "incremental"):
-            fillcolor = "aquamarine"
+        assert self.config is not None, self.unique_id
+        fillcolor = super(ModelNode, self).gvattrs["fillcolor"]
 
-        if (
+        more_than_100gb = (
             self.catalog_entry
             and self.catalog_entry.num_bytes  # type: ignore[attr-defined]
             and self.catalog_entry.num_bytes > 100_000_000_000  # type: ignore[attr-defined]
+        )
+
+        if self.config.materialized in ("table", "incremental"):
+            fillcolor = "aquamarine"
+
+        if self.config.materialized == "table" and more_than_100gb:
```

Comment:
I am confused, don't we want to override this if it *is* clustered/partitioned already?

Reply:
Ok, I'm actually going to combine these conditions. Technically a large table could be cheap to build but end up with many rows (e.g. huge cross join) which isn't _necessarily_ a problem.